airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml

```diff
@@ -23,6 +23,18 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         expect_records:
           path: "integration_tests/expected_records.jsonl"
+          extra_fields: no
+          exact_order: no
+          extra_records: yes
+        empty_streams:
+          - name: ticket_comments
+            bypass_reason: "unable to test due to fast-changing data"
+#          - name: ticket_metric_events
+#            bypass_reason: "unable to test due to fast-changing data"
+#          - name: tickets
+#            bypass_reason: "unable to test due to fast-changing data"
+#          - name: users
+#            bypass_reason: "unable to test due to fast-changing data"
```

Comment:
What's goin on here? Can we remove this?

Reply:
This was streams with wrong records. I just left them because their records did not match with fresh expected records (they become irrelevant on the next day). I updated records again but left streams commented just for next time to know what streams potentially may have problems